### PR TITLE
v7-logout: fix(plugin-credentials): Fixes issue with token details getting repopulated on logout

### PR DESCRIPTION
**Pull request by saurjai3**

### CODE REVIEW

File: packages/spark-core/src/spark-core.js [hunk 1]
```diff
@@ -223,7 +223,6 @@ const SparkCore = AmpState.extend({
         return Promise.resolve();
       })
       .then(() => Promise.all([
-        this.boundedStorage.clear(),
```
Comment (ianwremmel) [inline packages/spark-core/src/spark-core.js]: Why are you removing this?

File: packages/plugin-credentials/test/unit/spec/credentials.js [hunk 2]
```diff
@@ -101,6 +101,13 @@ describe(`plugin-credentials`, () => {
       });
     });
 
+    describe(`when logging out`, () => {
+      it(`rejects getUserToken`, () => {
+        spark.credentials.isLoggingOut = true;
```
Comment (ianwremmel) [inline packages/plugin-credentials/test/unit/spec/credentials.js]: Is there a test that asserts `isLoggingOut` gets set when logging out?

File: packages/plugin-credentials/src/credentials/common.js [hunk 3]
```diff
@@ -205,10 +205,10 @@ export default {
   logout() {
     return Promise.all(this.userTokens.map((token) => token.revoke()
       .catch((reason) => this.logger.warn(`credentials: token revocation failed for ${token.scope}, ignoring`, reason))))
-      .then(() => this.userTokens.reset())
+      .then(() => this.unset(`userTokens`))
```
Comment (ianwremmel) [inline packages/plugin-credentials/src/credentials/common.js]: What problem does this change solve?

**Comment by saurjai3** [comment]
opened a new PR: https://github.com/ciscospark/spark-js-sdk/pull/549
hence closing this one.

**Pull request closed without merging**